### ISSUE TITLE
Mentor Spirit: Death does not provide free Killing Hands or Plague Cloud

### ISSUE THREAD

**Issue by kazrakcom** [chummer5a/chummer5a#3524]
**Describe the bug**
Selecting the Mentor Spirit: Death does not provide the Killing Hands or Plague Cloud power for free.  Instead, it is provided at half cost (0.25pp instead of 0.5pp).

**To Reproduce**
Create a character with all default settings (Priority/Standard).
Use default priorities (Human, B in magic, Adept, 6 Magic).
On 'Common' tab, 'Add Quality'.  Select 'Mentor Spirit'.
Select 'Death' mentor spirit.
Select any skill.
Select either ability from popup, "Killing Hands" or "Plague Cloud".

**Expected behavior**
Selected ability added to 'Adept Powers' list, cost 0, delete button not visible.

**Screenshots**
![ChummerScreenshot](https://user-images.githubusercontent.com/623959/54889793-a0eb1300-4e63-11e9-9d4f-54eecaf643fd.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 5.205.0

**Comment by Gidoran** [comment]
I did some digging into this because it's relevant to a character I am building.

This issue stems from the Death mentor's bonuses being improperly built in the mentors.xml file. The issue is multifaceted; the way that it's asking people to select powers is broken and does not follow the same paradigm as other mentor spirits with multiple power selections (Berserker, for example) and instead uses some <selectpowers> tag which doesn't appear to add the bonuses properly.

Secondly, the Mage side of things has a todo and an empty <bonus> tag which when not commented out seems to factor in to the damn thing breaking.

I've tried to upload a file with a potential fix (It works for me, your mileage may vary, consult a doctor if etc) but since github isn't cooperating with me, have a gdrive link: https://drive.google.com/open?id=1OPLfDbWb-k3JFbNEvsIccFnzWc12e4Py

I don't advise using this but it should help whoever goes to fix this figure out an easy hack. Honestly I don't think the selectpowers code is worth working when this paradigm already exists and provides a more consistent user experience.
